src/tools/ScenarioMeasurement/Util/Logger.cs

```diff
@@ -15,6 +15,16 @@ public void Log(string message)
             Console.WriteLine(message);
         }
 
+        public void LogHeader1(string message)
+        {
+            Console.WriteLine($"=============== {message} ================ ");
+        }
+
+        public void LogHeader2(string message)
+        {
+            Console.WriteLine($"***{message}***");
+        }
+
```

Comment:
1) These names are not great, doesn't help explain what they do.
2) Instead of methods like this that hard code some number of = and *, why not a method that takes the string, defaults to a char to surround with, and a width? 
3) Do we really even need headers like this in the output? Maybe we ditch them.

Reply:
Good suggestion! 
1. I agree, but I couldn't think of a better naming. Any suggestion?
2. It seems easier to me to call ```logger.LogHeader1(<message>)```  than to call ```logger.LogHeader(<number of symbols>, <width>, <message>)``` multiple times?? 
3. Maybe it's just my personal feeling but the headers help me identify important info out of a great deal of texts very well, just like how BenchmarkDotNet renders its layout. I hope it's not a personal preference thing so I'll ask @DrewScoggins  what do you think?

Reply:
Can use default parameters for width and character. Or just hard code it to be some fixed width and put the string in the middle.

Reply:
Just want to clarify before I make the change, it is to get rid of the confusing ```LogHeader``` methods?

Reply:
Yes. I think we should not have `LogHeader1` and `LogHeader2`. Either define what makes them different (`LogHeader`, `LogWhatever`) or collapse them to one method. What do we use `=` and `*` to indicate? Do we need both?